### PR TITLE
pake: add page

### PR DESCRIPTION
**Pull request by fnfontana**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fnfontana** [comment]
Hello, I've added a new page for [Pake](https://github.com/tw93/Pake).

Note: I have read all the contributing guide and tried to follow the rules, but I'm still learning Git/GitHub, so if I've done something wrong, just let me know to fix it.

File: pages/common/pake.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# pake
+
+> Turn any webpage into a desktop app with Rust/Tauri.
+> More information: <https://github.com/tw93/Pake>.
+
+- Package a web page:
+
+`pake https://www.google.com/`
```
Comment (blueskyson) [inline pages/common/pake.md:8]:
```suggestion
`pake {{https://www.google.com/}}`
```

File: pages/common/pake.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Package a web page with specific window size:
+
+`pake --width 800 --height 600 "https://www.google.com/`
```
Comment (blueskyson) [inline pages/common/pake.md:12]:
```suggestion
`pake --width {{800}} --height {{600}} {{https://www.google.com/}}`
```

File: pages/common/pake.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Package a web page with a custom application name and icon:
+
+`pake --name "Google" --icon "path/to/icon.ico" https://www.google.com/`
```
Comment (blueskyson) [inline pages/common/pake.md:16]:
```suggestion
`pake --name {{Google}} --icon {{path/to/icon.ico}} {{https://www.google.com/}}`
```

File: pages/common/pake.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Package a web page with non-resizable window:
+
+`pake --no-resizable https://www.google.com/`
```
Comment (blueskyson) [inline pages/common/pake.md:20]:
```suggestion
`pake --no-resizable {{https://www.google.com/}}`
```

File: pages/common/pake.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Package a web page with fullscreen mode:
+
+`pake --fullscreen https://www.google.com/`
```
Comment (blueskyson) [inline pages/common/pake.md:24]:
```suggestion
`pake --fullscreen {{https://www.google.com/}}`
```

File: pages/common/pake.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Package a web page with transparent title bar:
+
+`pake --transparent https://www.google.com/`
```
Comment (blueskyson) [inline pages/common/pake.md:28]:
```suggestion
`pake --transparent {{https://www.google.com/}}`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution. Just left some syntax suggestions.

**Comment by fnfontana** [comment]
> Thanks for contribution. Just left some syntax suggestions.

All suggestions committed.

File: pages/common/pake.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# pake
+
+> Turn any webpage into a desktop app with Rust/Tauri.
+> More information: <https://github.com/tw93/Pake>.
+
+- Package a web page:
+
+`pake {{https://www.google.com/}}`
+
+- Package a web page with specific window size:
```
Comment (kbdharun) [inline pages/common/pake.md:10]:
```suggestion
- Package a web page with a specific window size:
```

File: pages/common/pake.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`pake --width {{800}} --height {{600}} {{https://www.google.com/}}`
+
+- Package a web page with a custom application name and icon:
+
+`pake --name {{Google}} --icon {{path/to/icon.ico}} {{https://www.google.com/}}`
+
+- Package a web page with non-resizable window:
```
Comment (kbdharun) [inline pages/common/pake.md:18]:
```suggestion
- Package a web page with a non-resizable window:
```

File: pages/common/pake.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`pake --no-resizable {{https://www.google.com/}}`
+
+- Package a web page with fullscreen mode:
+
+`pake --fullscreen {{https://www.google.com/}}`
+
+- Package a web page with transparent title bar:
```
Comment (kbdharun) [inline pages/common/pake.md:26]:
```suggestion
- Package a web page with a transparent title bar:
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. I have some minor suggestions for this page.

**Comment by fnfontana** [comment]
> Thanks for your contribution. I have some minor suggestions for this page.

Suggestions commited.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**